Editor/SettingsAnalysis/PlayerSettingsAnalyzer.cs

```diff
@@ -44,11 +44,42 @@ class PlayerSettingsAnalyzer : ISettingsModuleAnalyzer
             })
         };
 
+        static readonly Descriptor k_IL2CPPCompilerConfigurationMasterDescriptor = new Descriptor(
+            "PAS1004",
+            "Player: IL2CPP Compiler Configuration",
+            new[] { Area.BuildTime },
+            "<b>C++ Compiler Configuration</b> is set to <b>Master</b>. The build time will be longer. Keep this mode only for shipping builds.",
+            "To have optimal build time, change <b>Project Settings ➔ Configuration ➔ C++ Compiler Configuration</b> to <b>Release</b>.")
+        {
+            fixer = (issue => {
+                SetIL2CPPConfigurationToRelease();
+            }),
+
+            messageFormat = "Player : C++ Compiler Configuration is set to 'Master'. The build time will be longer."
+        };
+
+        private static readonly Descriptor k_IL2CPPCompilerConfigurationDebugDescriptor = new Descriptor(
+            "PAS1005",
+            "Player: IL2CPP Compiler Configuration",
+            new[] { Area.CPU },
+            "<b>C++ Compiler Configuration</b> is set to <b>Debug</b>. The performances will be suboptimal. Keep this mode only for debugging only.",
+            "To have optimal performances, change <b>Project Settings ➔ Configuration ➔ C++ Compiler Configuration</b> to <b>Release</b>.")
+        {
+            fixer = (issue =>
+            {
+                SetIL2CPPConfigurationToRelease();
+            }),
+
+            messageFormat = "Player : C++ Compiler Configuration is set to 'Debug'. The performances will be suboptimal."
```

Comment:
I would omit the " The performances will be suboptimal." as this is already in the description.